cli/cmd/plugin/unmanaged-cluster/test/e2e/utils/cli.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func InstallTCE() error {
+	wd, err := os.Getwd()
+	if err != nil {
+		log.Println("error while getting current working directory", err)
+	}
+
+	indTCE := strings.LastIndex(wd, "community-edition")
+	topDirPath := wd[0 : indTCE+17]
+	err = os.Chdir(topDirPath)
+	if err != nil {
+		log.Println("error while changing directory to the top:", err)
+		return err
+	}
+
+	// installing dependencies i.e docker and kubectl
+	err = runDeployScript("hack/ensure-deps/ensure-docker.sh")
+	if err != nil {
+		log.Fatal(err)
+		return err
+	}
+
+	err = runDeployScript("hack/ensure-deps/ensure-kubectl.sh")
+	if err != nil {
+		log.Fatal(err)
+		return err
+	}
+
+	// build.sh is creating a release and installing TCE for latest code
+	return runDeployScript("test/download-or-build-tce.sh")
```

Comment:
@dvonthenen can you confirm this will do a build of the current code and not download the daily build?